sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -52,6 +67,17 @@ public void testViewCanBeCreatedSelectedAndThenDropped() throws Exception {
         }
     }
 
+    @Test
+    public void testViewCanBeUsedForJoins() {
+        execute("CREATE TABLE t1 (x INTEGER, a STRING)");
+        execute("INSERT INTO t1 (x, a) VALUES (1, 'foo')");
+        execute("REFRESH TABLE t1");
+        execute("CREATE VIEW v1 AS select * FROM t1");
+        execute("CREATE VIEW v2 AS select * FROM t1");
+        assertThat(printedTable(execute("SELECT * FROM v1 INNER JOIN v2 ON v1.x = v2.x").rows()), is("foo| 1| foo| 1\n"));
+        execute("DROP VIEW v1, v2");
```

Comment:
Given the teardown, this shouldn't be necessary here?

Reply:
nope, not any more